apache/trafficcontrol
# Add a second origin server to CDN in a Box

## What does this PR (Pull Request) do?

This PR

- [x] is not related to any Issue 
- Renames the CDN in a Box `origin` server to `origin-01`
- Adds an `origin-02` server to CDN in a Box
- Enables the multi-site origin feature for CDN in a Box's `demo1` delivery service (was meant to be done in #5164)
- Fixes a bug where the Traffic Ops server objects for CiaB's origin server did not include an IPv6 address because the BusyBox implementation of the `route` command that Alpine uses by default does not include the `-6` flag. Fixed by installing the actual `route` command.

## Which Traffic Control components are affected by this PR?


- CDN in a Box
- Documentation
- Traffic Ops (v1 API tests)
- CI tests

## What is the best way to verify this PR?

- Run CDN in a Box, make sure the readiness container exits successfully
- Build the documentation
- Check the result of the *CDN-in-a-Box CI* workflow

## If this is a bug fix, what versions of Traffic Control are affected?

For the origin `route` command fix:

- master
- 5.0.0 (RC5)
- 4.1.1

## The following criteria are ALL met by this PR
